pkg/common/common.go

```diff
@@ -488,12 +489,17 @@ func GetVolumeMode(volUtil util.VolumeUtil, fullPath string) (v1.PersistentVolum
 }
 
 // NodeExists checks to see if a Node exists in the Indexer of a NodeLister.
-func NodeExists(nodeLister corelisters.NodeLister, nodeName string) (bool, error) {
-	_, err := nodeLister.Get(nodeName)
-	if errors.IsNotFound(err) {
-		return false, nil
+// It uses the well known label `kubernetes.io/hostname` to find the Node.
+func NodeExists(nodeLister corelisters.NodeLister, nodeLabelValue string) (bool, error) {
+	req, err := labels.NewRequirement(NodeLabelKey, selection.Equals, []string{nodeLabelValue})
```

Comment:
can you use the original method `_, err := nodeLister.Get(nodeName)` and if not found, use label as fallback, that would be safer, thx

Reply:
good point. I've modified the code to keep the original get